dallinger/command_line.py

```diff
@@ -535,7 +549,20 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1, exp_config=
     log("Launching the experiment on MTurk...")
 
     launch_request = requests.post('https://{}.herokuapp.com/launch'.format(app_name(id)))
-    launch_data = launch_request.json()
+    try:
```

Comment:
This code is almost identical to the code on line 337. Maybe we should abstract this into some kind of `launch` function that takes as input the `base_url`, and then use it both in `debug()` and `deploy_sandbox_shared_setup()`?